modules/nf-core/tidk/plot/tests/main.nf.test

```diff
@@ -0,0 +1,73 @@
+nextflow_process {
+
+    name "Test Process TIDK_PLOT"
+    script "../main.nf"
+    process "TIDK_PLOT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "tidk"
+    tag "tidk/plot"
+
+    test("homo_sapiens-genome_21_fasta-success") {
+
+        setup {
+            run("TIDK_SEARCH") {
+                script "../../../../nf-core/tidk/search"
+
+                process {
+                    """
+                    input[0] = [
+                        [ id:'test' ],
+                        file(params.test_data['homo_sapiens']['genome']['genome_21_fasta'], checkIfExists: true)
+                    ]
+                    input[1] = 'TTAGGG'
+                    """
+                }
+            }
+        }
+
+        when {
+            process {
+                """
+                input[0] = TIDK_SEARCH.out.tsv
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() },
+                { assert snapshot(process.out.versions).match("versions") }
+            )
+        }
+
+    }
+
+    test("stub") {
+
+        options '-stub'
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ],
+                    []
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert process.out.svg != null },
```

Comment:
Recent version (from yesterday) of `nf-core` linting will no longer fail if an empty file md5sum is found in a stub test, so this line could potentially be replaced by `{ assert snapshot(process.out).match() }` if you want. But this is minor, if you prefer the stub test as is, let's keep it as is!

Reply:
I have applied the change.